dev/io.openliberty.persistence.3.0.thirdparty/resources/org/eclipse/persistence/exceptions/i18n/BeanValidationExceptionResource.nlsprops

```diff
@@ -29,10 +29,10 @@
 # -------------------------------------------------------------------------------------------------
 7500=Bean validation mode on {0}marshaller is set to ON, but no Bean validation provider was found.
 
-7501=Bean validation mode on {0}marshaller is set to an unsupported value, "{1}".
+7501=Bean validation mode on {0}marshaller is set to illegal value, "{1}".
 
 7510=Constraints violated on {0}marshalled bean:
 {1}{2}
 
-7525=Cannot parse the {0} property, because it is both annotated with @NotNull and has the "xs:nillable=true" attribute.
+7525=Cannot parse property {0}, because it is both annotated with @NotNull and has attribute "xs:nillable=true".
```

Comment:
- Change `property {0}` back to `the {0} property`
- Change `has attribute "xs:nillable=true"` back to `has the "xs:nillable=true" attribute`